site/content/blogs/release-v127.en.md

```diff
@@ -48,6 +48,52 @@ or multiple listeners on different ports and protocols for [network load balance
 
 ## Sidecar improvements
 
+You can now build and push container images for sidecar containers just like your main container. Additionally, you can now specify the path to a local environment file for sidecar containers.
```

Comment:
```suggestion
You can now build and push container images for sidecar containers just like you can for your main container. Additionally, you can now specify the path to a local environment file for sidecar containers.
```

Reply:
It's one env file for all of the sidecar containers? If yes, leave as is; if no, change to:
```suggestion
You can now build and push container images for sidecar containers just like your main container. Additionally, you can now specify the paths to local environment files for sidecar containers.
```